testing/src/main/java/org/jdbi/v3/testing/JdbiRule.java

```diff
@@ -134,7 +134,7 @@ public JdbiRule withPlugin(final JdbiPlugin plugin) {
     }
 
     @Override
-    protected void before() throws Throwable {
+    public void before() throws Throwable {
```

Comment:
Strictly speaking, changing this from `protected` to `public` is a breaking change, since third-party implementors (if any) could already override it with `protected` access.

I think the fallout is minimal, but it's probably worth a mention in the release notes.